crates/core/app/src/lib.rs

```diff
@@ -19,7 +19,7 @@ pub use crate::{
 
 use once_cell::sync::Lazy;
 
-pub const APP_VERSION: u64 = 1;
```

Comment:
nit: the `TOTAL_HALT_COUNT` var has a friendly docstring. Can we add one here? Especially since this value isn't actually used anywhere (yet). But describing when and why it should be incremented is super important.

Reply:
Done, and squashed in.

Reply:
Thanks!!